### PR TITLE
Fixes to component styles

### PR DESCRIPTION
**Pull request by sheridanvk**
I think the tooltip component was build according to a bad implementation I'd done in the early days of CSS modules, and it was importing in the globals.styl file from the styles directory. This meant a bunch of styles that shouldn't have been available to the other components were available, so components looked fine in storybook when in fact they weren't self-composed.

Fixed a bunch of things but please let me know if you can see anything I missed! Everything is looking good at http://lumpy-organ.glitch.me/storybook

### CODE REVIEW

File: src/components/text/heading.styl [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
-
-.h4
-  font-size: 14px
\ No newline at end of file
+.heading
+  font-family: sansserif
+
+  &.h1
+    font-size: 22px
+  font-weight: bold
```
Comment (oliviacpu) [inline src/components/text/heading.styl]: I think this indentation is causing `font-weight: bold` to be applied to `.heading`

Comment (sheridanvk) [reply src/components/text/heading.styl]: good catch - fixed, thanks!

**Comment by taravancil** [comment]
Good catch on this, thanks!

File: src/components/text/markdown.styl [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 .markdownContent
   text-rendering: optimizeLegibility
+  font-family: sansserif
```
Comment (ttseng) [inline src/components/text/markdown.styl]: hey!  just noticed there is a small regression here where markdownContent shouldn't default to sansserif (it should be shown with menlo font if it's editable):

image

I'm going to do a more thorough check on styling to see if I can find anything else and will put that in its own PR.

**Comment by sheridanvk** [comment]
thanks Tiff, good catch! I think it is monospace after clicking in, but not
before the click. Is that the behaviour you're going to fix as well? I
guess we need to add the .content-editable class earlier.

**Comment by sheridanvk** [comment]
Ahh that's actually the problem - there is no content-editable class on
Markdown. It was incorrectly inheriting its styles from global.styl acting
on its parent before - we should be passing the .content-editable class
from EditableDescription down into Markdown so we can change the font
accordingly.

**Pull request merged**